GitOut/Features/Git/Stage/GitStagePage.xaml

```diff
@@ -362,6 +388,83 @@
+                </Grid>
+                <ListView
+                    Grid.Row="4"
+                    ItemsSource="{Binding AmendFiles}"
+                    SelectedValue="{Binding SelectedAmendChange}"
+                >
+                    <ListView.Style>
+                        <Style TargetType="{x:Type ListView}" BasedOn="{StaticResource GitChangeListStyle}">
+                            <Setter Property="Visibility" Value="Collapsed"/>
+                            <Style.Triggers>
+                                <DataTrigger Binding="{Binding DataContext.AmendLastCommit, RelativeSource={RelativeSource AncestorType={x:Type local:GitStagePage}}}" Value="True">
+                                    <Setter Property="Visibility" Value="Visible"/>
+                                </DataTrigger>
+                            </Style.Triggers>
+                        </Style>
+                    </ListView.Style>
+                    <ListView.ItemContainerStyle>
+                        <Style TargetType="{x:Type ListViewItem}">
+                            <Setter Property="Height" Value="22"/>
+                            <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+                            <Setter Property="Foreground" Value="{DynamicResource MaterialForegroundBase}"/>
+                        </Style>
+                    </ListView.ItemContainerStyle>
+                    <ListView.ItemTemplate>
+                        <DataTemplate DataType="{x:Type files:GitFileViewModel}">
+                            <Grid
+                                HorizontalAlignment="Stretch"
+                                VerticalAlignment="Stretch"
+                            >
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="Auto"/>
+                                    <ColumnDefinition Width="*"/>
+                                </Grid.ColumnDefinitions>
+                                <Canvas Width="24" Height="24" Margin="0 0 3 0">
+                                    <Canvas.LayoutTransform>
+                                        <ScaleTransform ScaleX=".6" ScaleY=".6" />
+                                    </Canvas.LayoutTransform>
+                                    <Path
+                                        Fill="{Binding Status, Converter={StaticResource DiffTypeToBrushConverter}}"
+                                        Data="{Binding IconResourceKey, Converter={converters:StaticResourceConverter}}"
+                                    />
+                                </Canvas>
+                                <TextBlock
+                                    Grid.Column="1"
+                                    Style="{StaticResource NormalTextStyle}"
+                                    Text="{Binding FileName.Name}"
```

Comment:
we use `DisplayName` for index and `Staged`, we can use that here as well to get the same path convention
```suggestion
                                    Text="{Binding DisplayName}"
```